src/V3Ast.h

```diff
@@ -1665,7 +1665,8 @@ class AstNode VL_NOT_FINAL {
     string origNameProtect() const;  // origName with --protect-id applied
     string shortName() const;  // Name with __PVT__ removed for concatenating scopes
     static string dedotName(const string& namein);  // Name with dots removed
-    static string prettyName(const string& namein);  // Name for printing out to the user
+    // Name for printing out to the user
+    static string prettyName(const string& namein) VL_MT_SAFE;
```

Comment:
```suggestion
    static string prettyName(const string& namein) VL_PURE;
```

(Most but not all static functions are likely pure)